WordPress/Classes/Models/Blog+Editor.swift

```diff
@@ -39,7 +39,7 @@ extension Blog {
     /// The editor to use when creating a new post
     ///
     var editor: MobileEditor {
-        return mobileEditor ?? GutenbergSettings().getDefaultEditor(for: self)
+        return mobileEditor ?? GutenbergSettings(database: UserDefaults.standard).getDefaultEditor(for: self)
```

Comment:
👋 @aerych - I wonder if we should update the [convenience init](https://github.com/wordpress-mobile/WordPress-iOS/blob/ef1794962ba5bdcc1becfa251941ef115043cd43/WordPress/Classes/Utility/Editor/GutenbergSettings.swift#L46) to use the singleton so that all callers can benefit.

I'm drawing a blank on why a fresh `UserDefaults()` would be needed?

Reply:
I'm glad to hear the way UserDefaults() is called is a mystery to more than just me.  I made the change here for the lighter touch but I think it's probably safe (and more helpful) to make move the assignment into GutenbergSettings.  To be slightly cleaner, I think we can get rid of the convenience initializer and make UserDefaults.standard a default parameter value in the  `init(database:)`.

Reply:
Good call on the default param. 👍